### PR TITLE
Neaten much of the HTML

### PR DESCRIPTION
**Pull request by jamierocks**

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -26,15 +26,16 @@
...
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="description" content="">
     <meta name="author" content="">
 
     <title>Sponge - Minecraft Modding API</title>
-    <link rel="icon" type="image/png" href="https://forums-cdn.spongepowered.org/uploads/default/original/2X/9/9ba706a80e45cf427617525ee2a19fad7bb6b109.png">
+
+    <link href="https://forums-cdn.spongepowered.org/uploads/default/original/2X/9/9ba706a80e45cf427617525ee2a19fad7bb6b109.png"
+          type="image/png" rel="icon">
```
Comment (kashike) [inline templates/base.html]: :-1: this makes it worse imo

Comment (progwml6) [inline templates/base.html]: i'd keep this on one line

File: templates/base.html [hunk 2]
```diff
@@ -164,13 +186,20 @@
...
+                }, i[r].l = 1 * new Date();
+        a = s.createElement(o),
+                m = s.getElementsByTagName(o)[0];
+        a.async = 1;
+        a.src = g;
+        m.parentNode.insertBefore(a, m)
+    })(window, document, 'script', 'https://www.google-analytics.com/analytics.js', 'ga');
+
+    ga('create', 'UA-59476017-1', 'auto');
+    ga('send', 'pageview');
```
Comment (progwml6) [inline templates/base.html:158]: this should probably be left alone

Comment (jamierocks) [inline templates/base.html:158]: #BlameIntelliJ

**Comment by progwml6** [comment]
@jamierocks this has merge conflicts

**Comment by jamierocks** [comment]
Yep I'll pull in the recent changes and confirm to the above changes

**Pull request merged**